api_v3/lib/KalturaResponseCacher.php

```diff
@@ -306,6 +307,47 @@ protected function isAnonymous($ks)
 		return false;
 	}
 
+	private static function handleCacheBasedServiceActions(&$params)
+	{
+		if (isset($params['service']) && isset($params['action']))
+		{
+			$service = $params['service'];
+			$action = $params['action'];
+			if ($service === 'session' && $action === 'start')
```

Comment:
In order to have unified look of the code , please move session start to a file of its own same as vote\Action